lib/stm32/f0/rcc.c

```diff
@@ -43,6 +43,7 @@
 /* Set the default clock frequencies */
 uint32_t rcc_ahb_frequency = 8000000; /* 8MHz after reset */
 uint32_t rcc_apb1_frequency = 8000000; /* 8MHz after reset */
+uint32_t rcc_lse_frequency = 32768;		/* most common default */
```

Comment:
I'd kinda just simply not support automatically setting up baud rates if you've configured the source to be LSE.  It's an edge case I think, and requiring this sort of variable to exist?